### ISSUE TITLE
(32-bit) FETCH ALL -> FETCH 2147483647

### ISSUE THREAD

**Issue by df7cb** [lelit/pglast#142]
On 32-bit i386 Debian unstable:

```
$ echo "fetch all from foo" | pgpp
FETCH FORWARD 2147483647 foo
```

Seen with libpg-query 15+pglast 5.6 as well as libpg-query 16 + pglast 6dev0.

This isn't caught by regression tests.

(64-bit yields the expected `FETCH ALL foo`)

**Comment by lelit** [comment]
Thanks for the report! IIRC, that `ALL` is encoded as a particular integer value... will check!

**Comment by lelit** [comment]
Yes, this is indeed an oversight in the generation of the `enums` module.

I think I cured the problem, CI will hopefully tell. I will integrate this in v6 soon [if it works].

**Comment by lelit** [comment]
I just released v5.8 with this fix.
